api/integrations/fifaIndexWebScrapper.js

```diff
@@ -68,6 +58,10 @@ const TYPE_OF_TEAMS = {
     WOMEN_INTERNATIONAL : 2
 }
 
+/* Since this is a support script,
+we should not be so hard on the no-console rule.*/
+const Console = console;
```

Comment:
alright. but why this Console = console ?

Reply:
YOU ALREADY ASKED THIS BRO

Reply:
My internet yesterday was terrible :/